009-ui-and-admin-server-security.md

```diff
@@ -0,0 +1,84 @@
+# Security Extension
+
+This proposal is supplementary to the security documentation in the [Using Strimzi](https://strimzi.io/docs/operators/latest/using.html#security-str) guide.
+
+## Problem description
+Strimzi currently supports SCRAM-SHA-512, Oauth2 and Mutual TLS for authentication of clients connecting to the Kafka brokers.
```

Comment:
```suggestion
Strimzi currently supports SCRAM-SHA-512, OAuth2 and Mutual TLS for authentication of clients connecting to the Kafka brokers.
```

Reply:
I believe I have fixed all uses of the term OAuth2.